.env.test

```diff
@@ -0,0 +1,19 @@
+REDIS_URL=redis://localhost
```

Comment:
I think, you need add this file to gitignore, because your config can differ from others.
But add dotenv its great, I thought about it too.